src/NativeAdComponents.js

```diff
@@ -146,6 +146,61 @@ export const MediaView = (props) => {
   if (!nativeAdView) return null;
 
   return (
-    <View {...props} ref={viewRef}/>
+    <View {...props} ref={viewRef} />
+  );
+};
+
+export const StarRatingView = (props) => {
+  const {style, ...otherProps} = props;
+
+  const maxStarCount = 5;
+  const starColor = style.color ?? "#ffe234";
+  const starSize = style.fontSize ?? 10;
+
+  const {nativeAd, nativeAdView} = useContext(NativeAdViewContext);
+
+  const FilledStar = useCallback(({size,color}) => {
```

Comment:
Is it necessary to use `useCallback(...)` as well? Could we do:

```
  const FilledStar = ({ size, color }) => (
    // black star in unicode
    <Text style={{ fontSize: size, color }}>{String.fromCodePoint(0x2605)}</Text>
  );
```

Reply:
Ok, removed.  It is not necessary.